x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/file/FileRealmSettings.java

```diff
@@ -14,6 +14,7 @@
 
 public final class FileRealmSettings {
     public static final String TYPE = "file";
+    public static final String NAME = "default_file";
```

Comment:
As above.
Given that an admin can be name these realms whatever they want, I think `NAME` has the potential to be misleading to future readers.

Reply:
You're right, it was confusing for me too, I've renamed it to `DEFAULT_NAME`.